app/uk/gov/hmrc/apinotificationpull/model/Notifications.scala

```diff
@@ -23,3 +23,8 @@ case class Notifications(notifications: List[String])
 object Notifications {
   implicit val notificationsJF = Json.format[Notifications]
 }
+
+object JsErrorResponse {
+  def apply(message: String) =
```

Comment:
think about using native Scala DSL for creating XML here - it will ensure that XML is well formed. See https://github.com/hmrc/customs-api-common/blob/master/app/uk/gov/hmrc/customs/api/common/controllers/ErrorResponses.scala#L67